website/project/views/node.py

```diff
@@ -596,6 +597,21 @@ def remove_private_link(*args, **kwargs):
         link = PrivateLink.load(link_id)
         link.is_deleted = True
         link.save()
+
+        for node in link.nodes.all():
+            log_dict = {
+                'project': node.parent_id,
```

Comment:
[Seems to be the _id?](https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/node.py#L967). Also, I was following the example of [other node logs that use node.parent_id](https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/node.py#L1081)